components/waitwhile/app/waitwhile.app.ts

```diff
@@ -1,13 +1,197 @@
 import { defineApp } from "@pipedream/types";
+import api from "api";
 
 export default defineApp({
   type: "app",
   app: "waitwhile",
-  propDefinitions: {},
+  propDefinitions: {
+    addTag: {
+      label: "Add Tag",
+      type: "string",
+      optional: true,
+      description: "Adds specified tag to existing tags",
+    },
+    customerId: {
+      label: "Customer ID",
+      type: "string",
+      description: "Identifier of customer, automatically derived from visitor contact information if not provided.",
+    },
+    customerNoteId: {
```

Comment:
Would be nice to add `async options()` for this prop

Reply:
Can a property have more than one async options?  Theses props are passed to several different functions so I would need an async option for each function.